servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/AsyncContextProvider.java

```diff
@@ -66,18 +66,28 @@ interface AsyncContextProvider {
     Scope attachContextMap(ContextMap contextMap);
 
     /**
-     * Capture the current context with the provided {@link ContextMap} as the captured {@link AsyncContext} state.
+     * Capture the current context with the provided {@link ContextMap} as the captured {@link AsyncContext} state
+     * using sharing semantics, if applicable.
      * @return the captured context to be restored across async boundaries.
+     * @deprecated this should be removed once we remove the `*SetContextOnSubscribe` operators.
      */
-    CapturedContext captureContext(ContextMap contextMap);
+    @Deprecated
```

Comment:
Please add a typical `// FIXME: 0.43` prefix to help us catch it consistently with other changes